buildarr/config/base.py

```diff
@@ -814,49 +814,39 @@ def yaml(
             YAML representation of the model
         """
         return yaml.safe_dump(  # type: ignore[call-overload]
-            json.loads(self.json(*args, **kwargs)),
+            json.loads(self.model_dump_json(*args, **kwargs)),
```

Comment:
Consider switching to `model_dump` and passing the resulting `dict` directly to `yaml.safe_dump`.

Reply:
Changed to `model_dump(mode="json")`. Appears to work properly.